src/js_native_api.h

```diff
@@ -1,8 +1,8 @@
 #ifndef SRC_JS_NATIVE_API_H_
 #define SRC_JS_NATIVE_API_H_
 
-#include <stddef.h>
-#include <stdbool.h>
+#include <stddef.h>   // NOLINT
```

Comment:
```suggestion
#include <stddef.h>   // NOLINT(modernize-deprecated-headers)
```
etc...

Reply:
Run into lint error `Unknown NOLINT error category: modernize-deprecated-headers  [readability/nolint] [5]`

https://travis-ci.com/nodejs/node/jobs/189093755

Reply:
🤔 ...
I'l look into patching `cpplint.py`

Reply:
You can use `clang-tidy --list-checks -checks='*'` to see the clang-tidy rules.

Also, can you remove the `dont-land` label.

I still need to fix some comment format issue in this pr, hope I can get it done this evening.